### DISCUSSION TITLE
After update with 1.32.1, it gives this error, can I recover this near node

### DISCUSSION THREAD

**Discussion by HarishankarVj12** [near/nearcore#8816] [Q&A]
thread 'main' panicked at 'start_with_config: unable to open database at /root/.near/data
Caused by:
    Database version 35 is created by a newer version of neard, please update neard', neard/src/cli.rs:517:18
stack backtrace:
   0: rust_begin_unwind
             at ./rustc/d5a82bbd26e1ad8b7401f6a718a9c57c96905483/library/std/src/panicking.rs:575:5
   1: core::panicking::panic_fmt
             at ./rustc/d5a82bbd26e1ad8b7401f6a718a9c57c96905483/library/core/src/panicking.rs:64:14
   2: core::result::unwrap_failed
             at ./rustc/d5a82bbd26e1ad8b7401f6a718a9c57c96905483/library/core/src/result.rs:1791:5
   3: neard::cli::RunCmd::run::{{closure}}
   4: tokio::task::local::LocalSet::run_until::{{closure}}
   5: neard::cli::NeardCmd::parse_and_run
   6: neard::main
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.

**Comment by TatyOko28** [comment]
This error occurs because your node's database version (35) was created by a newer version of neard. The error suggests that the current version of neard is outdated for the existing database.
Solution: Update your neard binary to the latest compatible version, or reset the database if you cannot update.
